packages/web/app/edge/routing/route.ts

```
@@ -366,20 +367,23 @@
+    for (const [service, _baseEndpoint] of Object.entries(
+      config?.endpoints || {}
+    )) {
       endpoints[service] = getOptimalEndpoint(
         service as keyof RoutingConfig['endpoints'],
         userContext.tier,
-        userContext.region,
+        userContext.region
       )
     }
 
+    const freeConfig = ROUTING_CONFIGS.free!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)